pootle/apps/pootle_store/models.py

```diff
@@ -998,7 +998,9 @@ class Store(models.Model, CachedTreeItem, base.TranslationStore):
 
     class Meta(object):
         ordering = ['pootle_path']
-        unique_together = ('parent', 'name')
+        unique_together = [
+            ('parent', 'name'),
+            ["obsolete", "translation_project", "tp_path"]]
```

Comment:
I have the same doubt re `obsolete`.

Nit: Use a tuple, since as you can see it is what is being used in the migration.